sections/performance/nativeoverutil.md

```diff
@@ -5,7 +5,7 @@
 
 ### One Paragraph Explainer
 
-Sometimes, using native methods is better than requiring `lodash` or `underscore` because it will not lead in a performance boost and use more space than necessary.
+Sometimes, using native methods is better than requiring `lodash` or `underscore` because it will not lead to performance loss or take up more space than needed.
```

Comment:
Hi @germangamboa95 
Couldn't we rephrase this sentence to avoid the negation? I think the `it will not lead` makes the sentence a bit harder to understand.

Something like `Sometimes, using native methods is better than requiring _lodash_ or _underscore_ because those libraries can lead to performance loss or take up more space than needed` ?
I'm not set on that exact sentence, just clarifying what I mean :)